third_party/terraform/utils/provider_handwritten_endpoint.go.erb

```diff
@@ -4,7 +4,7 @@ package google
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/hashicorp/terraform-provider-google-beta/google-beta"
+	"github.com/hashicorp/terraform-provider-google-beta/v3/google-beta"
```

Comment:
Am I understanding correctly that this was referring to an older version of the beta provider because we weren't following semantic versioning?

Reply:
yes, technically. It was working nonetheless because it was all local to the module and had the same path as the module, but once the downstream PRs are merged the module name will include the `v3` and any references to the old module name will result in version 1.20 being downloaded and used by those imports.